toolkit/scripts/check_required_kernel_configs.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Script to check for required kernel configs in a kernel config file
+# Usage: python3 check_required_kernel_configs.py --required_configs <path to json of required configs> --config_file <path to config being checked>
+
+import json
+import argparse
+import sys
+
+def check_config_arch(input_file):
+    with open(input_file, 'r') as file:
+        contents = file.read()
+    file.close()
+    if "Linux/x86_64" in contents:
+        return "AMD64"
+    elif "Linux/arm64" in contents:
+        return "ARM64"
+    else:
+        return None
+
+def check_strings_in_file(json_file, input_file):
+    arch = check_config_arch(input_file)
+    with open(json_file, 'r') as file:
+        data = json.load(file)
+        configData = data['required-configs']
+
+    with open(input_file, 'r') as file:
+        contents = file.read().split("\n")
+
+    missing_configs = {}
+
+    for key, value in configData.items():
```

Comment:
Please use more descriptive names for the key and value variables. Otherwise it's getting somewhat hard to read the code.